### ISSUE TITLE
Missing options parameter in Auth type declaration

### ISSUE THREAD

**Issue by azzazkhan** [firebase/firebase-js-sdk#5072]
### [REQUIRED] Describe your environment

  * Operating System version: Windows 10 Home (64-bit)
  * Browser version: Chrome version 91.0.4472.114 (Official Build) (64-bit)
  * Firebase SDK version: v8.6.8
  * Firebase Product: auth

### [REQUIRED] Describe the problem
I was using a fix to suppress the auth warning banner described in [this comment](https://github.com/firebase/firebase-tools/issues/2773#issuecomment-744497705) in javascript but I recently switched to **typescript** and when I applied that fixed (passed the options object) the typescript compiled yelled at me that the `auth.useEmulator` expects one argument.

#### Steps to reproduce:
Using Next.js framework with Typescript and installed the `firebase` package using **yarn** package manager. Wrote the code and VS Code's built-in typescript compiler gave this **TypeError**.

#### Relevant Code:

```typescript
import firebase from "firebase/app";
import "firebase/auth";

const auth = firebase.auth();

// Expected 1 arguments, but got 2. ts(2554)
auth.useEmulator("http://localhost:9099", {
  disableWarnings: true
});
```

**Comment by azzazkhan** [comment]
Even though the updates were merged in this [PR #4119](https://github.com/firebase/firebase-js-sdk/pull/4119/files/65ad84ade8976061d452157963777158721cb18d#diff-e55586f73849d83d4c5c9395d01435088411e942b87ab62e2713ea18f1c64da9) in [`packages-exp/auth-types-exp/index.d.ts`](https://github.com/firebase/firebase-js-sdk/blob/65ad84ade8976061d452157963777158721cb18d/packages-exp/auth-types-exp/index.d.ts) types declaration file but the compiler is not picking types from that file.

**Comment by looptheloop88** [comment]
Hi @azzazkhan, thanks for the report. Per checking, this is a duplicate of #4591.

There's already an internal bug filed for this issue, however we can't provide definite timelines or details for when it will be available. You may keep an eye out on this thread or our [release notes](https://firebase.google.com/support/release-notes/js) to get updates in the future.
